### ISSUE TITLE
Execution failed for task ':flutter_callkit_incoming:compileDebugKotlin'.

### ISSUE THREAD

**Issue by francisnatty** [GetStream/stream-video-flutter#795]
You updated the package 5days  ago. but am still getting this error Execution failed for task ':flutter_callkit_incoming:compileDebugKotlin'. i thought it was a fix

**Comment by chung92726** [comment]
I am getting the same error did you fix it ?

**Comment by francisnatty** [comment]
> I am getting the same error did you fix it ?

Yes..I forked and edited the package.. https://github.com/francisnatty/flutter_callkit_incoming

**Comment by tisral** [comment]
Thanks @francisnatty.
 
pubspec.yaml
____________

`dependencies:
  flutter:
    sdk: flutter
  # Other dependencies...

  flutter_callkit_incoming: # Add a placeholder for version, though it will be overridden
    git:
      url: https://github.com/francisnatty/flutter_callkit_incoming
      ref: master`

**Comment by deven98** [comment]
Just an update here folks - we are working with the package creator to fix this issue and will update the SDK once it is done.

**Comment by Brazol** [comment]
I am closing this issue as the latest release (v0.7.0) has resolved the problem.
